docs/DeveloperGuide.adoc

```diff
@@ -0,0 +1,665 @@
+= A.T.A.S (Amazing Task and Assignment System) Developer Guide
+:site-section: UserGuide
+:toc:
```

Comment:
Would numbering the table of content help the reader with the flow of the documentation?